editor/src/components/context-menu-wrapper.tsx

```diff
@@ -291,12 +289,13 @@ export const MenuProvider = ({
         return
       }
 
-      show(
-        event,
-        localXHack_KILLME === 'local-x-coord-KILLME'
-          ? { position: { x: event.nativeEvent.offsetX, y: event.nativeEvent.pageY } }
-          : undefined,
-      )
+      show({
+        event: event,
+        position:
+          localXHack_KILLME === 'local-x-coord-KILLME'
```

Comment:
@benwolfram are we tracking this? we should really kill this localXHack!
this was meant as a temporary fix in a demo deadline push a few weeks ago, simply put here because we only had five minutes to fix the offset.

Reply:
Nope, this wasn't on my todo list. I am not even sure what it is tbh, but I can take a look here soon in a future PR, as I was intending to wrap work on Context Menu.